.github/workflows/build.yml

```diff
@@ -56,6 +56,8 @@ jobs:
           cargo build --verbose --color always --features rpc-client
           cargo build --verbose --color always --features rpc-client,rest-client
           cargo build --verbose --color always --features rpc-client,rest-client,tokio
+          cd ../lightning
+          cargo build --verbose --color always --features hashbrown
```

Comment:
We should also test with `hashbrown` in the "! matrix.build-net-tokio" block immediately above this one.